db/migrate/20161128201646_create_quote_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateQuoteLikes < ActiveRecord::Migration
+  def change
+    create_table :quote_likes do |t|
+      t.references :user, index: true, foreign_key: true, null: false
+      t.references :quote, index: true, foreign_key: true, null: false
+
+      t.timestamps null: false
+    end
```

Comment:
Probably also wanna copy existing data from the `votes` table, like I did when migrating to `post_likes` or `review_likes` (see their migrations for example code to copy+paste)